api/swagger.yml

```diff
@@ -5159,6 +5159,66 @@ paths:
         default:
           $ref: "#/components/responses/ServerError"
 
+  /repositories/{repository}/metadata/object/{type}/{object_id}:
+    parameters:
+      - in: path
+        name: repository
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: object_id
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: type
+        required: true
+        schema:
+          type: string
+          enum:
+            - range
+            - meta_range
+    get:
+      tags:
+        - internal
+      operationId: getMetadataObject
+      summary: return a lakeFS metadata object by ID
+      parameters:
+        - in: query
+          name: presign
+          required: false
+          schema:
+            type: boolean
+      responses:
+        200:
+          description: object content
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+          headers:
```

Comment:
? should return ETag header in response?

Reply:
`object_id`s are their own ETag (i.e. ranges and metaranges are immutable) - we don't keep this metadata anywhere and `BlockAdapter`s don't expose this.